### DISCUSSION TITLE
@esfx/internal-deprecate version 1.0.0-pre.23 break the react project build

### DISCUSSION THREAD

**Discussion by CiprianDraghici** [esfx/esfx#27] [Q&A] [answered]
Hello !

I identified something wrong with my ReactJS project build. Before the version mentioned above, all the things worked fine. The latest version breaks my project build with the following error:

![image](https://user-images.githubusercontent.com/11082863/136236532-921820d8-1bb9-4fec-be1f-b2699276ad17.png)

Can you provide me some information why this happened? 
I mention that my project is created with react-scripts-ts. The webpack config file is hidden and is not manipulated by me.
Should I add some special loader to handle this error ? 

Thanks in advanced,
Ciprian.

**Comment by rbuckton** [comment, marked as answer]
Bindingless `catch` has been part of the standard for several years now. It's possible the version of webpack or babel you are using is out of date. I can look into releasing a patch for internal-deprecate if this continues to be an issue.

**Reply by CiprianDraghici** [reply]
The React project is built with react-scripts-ts ^3.1.0
Webpack version is 4.46.0
Babel version is 7.2.0

**Reply by rbuckton** [reply]
I'm not certain when babel added support for bindingless `catch`, but it has been part of the standard since ES2019. It seems to parse fine in the BabelJS REPL on v7.15. Its a small enough change the `esfx` build target to ES2018 though, since that was the only syntax change in ES2019.

**Reply by rbuckton** [reply]
I published an update to work around this as `1.0.0-pre.24`.

**Reply by CiprianDraghici** [reply]
It's working fine with 1.0.0-pre.24 version. Thanks a lot @rbuckton !
